nengo/synapses.py

```diff
@@ -212,6 +212,18 @@ class LinearFilter(LinearSystem, Synapse):
         `scipy.signal.cont2discrete` for information about the options.
 
         .. versionadded:: 3.0.0
+    x0 : array_like, optional
+        Initial values for the system state. The last dimension must equal the
+        ``state_size``.
+
+        .. versionadded:: 3.1.0
+    initial_output : Distribution or float or (n_synapses,) array_like, optional
+        Initial output value(s) represented by the synapses. ``n_synapses`` is typically
+        equal to the connection's ``size_out``, except when ``post_obj`` is an
+        ``Ensemble`` and ``solver.weights`` is set, in which case ``n_synapses`` equals
+        the number of neurons in the post ``Ensemble``.
+
+        .. versionadded:: 3.1.0
```

Comment:
So these are both parameters from base classes, which is why I didn't document them here. I can understand the desire to document them here, particularly because they're passed explicitly rather than as part of a ``**kwargs``. This is especially helpful because we've got diamond inheritance here, so it's extra confusing for a user where to look if they want documentation for these arguments. The downside, though, is then we have duplication of docstrings. I'm wondering if it's possible to make some sort of note (ideally both here and in the original docstrings) so that if one of them changes, we know to update the other location. I'm not sure how best to do that, though.

Reply:
Could something like https://sublime-and-sphinx-guide.readthedocs.io/en/latest/reuse.html be used to define and reuse the same description in multiple places? There could also be some sort of 'inherit params' magic annotations. Seems like a fairly general problem that we wouldn't be the first to encounter.